### DISCUSSION TITLE
#savechromiecraftPvP!

### DISCUSSION THREAD

**Discussion by Spankthis** [chromiecraft/chromiecraft#6283] [General]
We need people start wondering how we can save the PvP-comminity on this server otherwice we start loosing some nice members. 
People are wanting to do battlegrounds, but they are filled either highly geared people what owns people in seconds or battlegrounds never popping. 

This is sad and we are really in need for changes in CC to get PvP-scene working and fun. 
I don't mind the honor amount needed for items, but it needs reworked to keep it rewarding and wanting to keep doing it. 

Arenas:
Almost dead

Battlegrounds:
Almost completely dead

WorldPvP:
No.

Lots of people are wanting to PvP-while gaming here, but now server is more giving penalty to PvP-players than anything. 
How we can start fixing this situation?

Lots of players in discord already pointed out that WotLK does not reward players on PvP that much than in vanilla/tbc. 

Not rewarding to do PvP for PvE stuff. 
Is it that we need to add some rewards of winning?
Tradeskill goods? 
Honor to get gems?
Reward bags?
PvE Items with honor?

BRAINSTORM HERE! 

WE NEED SAVE PvP IN THIS SERVER! 
PvP would bring more people, more people would mean more reports, more people would mean more PvP.

**Comment by Spankthis** [comment]
I would suggest myself:
Bring PvE-tradeskill items to be tradeable with honor or winners get random sack of tradeskill items. 

This would give PvP'ers more PvE rewards = Less grinding outside of PvP = More people in PvP 
More PvE players in PvP = More PvE rewards = More PvP

**Comment by Spankthis** [comment]
Posting some screenshots of ideas and suggestions:

![image_2023-10-25_181034286](https://github.com/chromiecraft/chromiecraft/assets/148990385/05c08ea6-32b9-43f8-b110-927c9690358b)


![image_2023-10-25_181210544](https://github.com/chromiecraft/chromiecraft/assets/148990385/265547a1-1bfb-4045-8bb1-e4ebfbf9c55b)

![image_2023-10-25_181245844](https://github.com/chromiecraft/chromiecraft/assets/148990385/dc8636ab-8e80-43f5-8507-7607b41e473d)


![image](https://github.com/chromiecraft/chromiecraft/assets/148990385/8e2438f4-0f03-4012-9758-6ec62c7c6dc3)


![image](https://github.com/chromiecraft/chromiecraft/assets/148990385/091d5a6c-709e-481c-8bb6-92a634df7db6)

**Reply by Spankthis** [reply]
![image](https://github.com/chromiecraft/chromiecraft/assets/148990385/46833661-942e-47cd-977e-c98728bde754)

**Comment by Gromphh** [comment]
BGs popped more often during 60 bracket because of the don julio band rings.  So i would suggest we have teir tokens be available for purchase via honor (something like max honor for half of a token)

**Comment by shaszun** [comment]
https://github.com/chromiecraft/chromiecraft/discussions/6046

**Comment by Never6** [comment]
fix bugs, what bugs you may ask? https://github.com/azerothcore/azerothcore-wotlk/issues/11719 (works with healthstones too) https://github.com/chromiecraft/chromiecraft/issues/4927 https://github.com/chromiecraft/chromiecraft/issues/3606 https://github.com/chromiecraft/chromiecraft/issues/2605 (never fixed happens with players all the time even if it doesn't show visually this means i could either lose some of an opener cause someones back turned and i wouldnt know or i could lose it due to his back turning into a wall) and in general servers core seems very laggy. I have no issues determining what is Line of Sight or What is back on any other server but this one.
add premades
add dmg reduction to match level 80 numbers (but leave crit chance alone) and only make it apply to players at or above 200 resil so you dont have people run around in full pve gear and esentially wear 2 gear sets. So if a player had 400 resil from gear he would get less global damage reduction because he doesnt need as much to add up to the 40% dmg reduction that 1k resil at 80 has (dunno if its actually 40% just using it as an example) but someone with 200resil needs more.

**Reply by Spankthis** [reply]
This could be tweaked to be even more starter friendly by buffing the whole resilience.
So newcomers with 70/100 resi would get more damange reduction already and it would be more welcoming to join with only reputation PvP-gear. 

So the your 40% damage reduction would be there around 420~ resi or 400~ 
100 resi would have 10%.

Lots of newcomers are complaining about getting singleshot and making 0 damage to anyone.
PvP is not welcoming in 3.3.5, but in small server like this... I think it should be better.

So what you think about Resi actually gets scaled just higher than it does now?
